src/components/templates/Auditorium/components/SectionPreview/SectionPreview.tsx

```diff
@@ -0,0 +1,73 @@
+import React from "react";
+import classNames from "classnames";
+import { useHistory } from "react-router";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faUserFriends, faLock } from "@fortawesome/free-solid-svg-icons";
+
+import { DEFAULT_SECTION_PREVIEW_TITLE } from "settings";
+
+import { useVenueId } from "hooks/useVenueId";
+import { useSectionSeatedUsers } from "hooks/auditoriumSections";
+
+import { UserList } from "components/molecules/UserList";
+
+import { AuditoriumSection } from "types/auditorium";
+
+import { WithId } from "utils/id";
+
+import "./SectionPreview.scss";
+
+export interface SectionPreviewProps {
+  section: WithId<AuditoriumSection>;
+}
+
+export const SectionPreview: React.FC<SectionPreviewProps> = ({ section }) => {
+  const { isLocked, title } = section;
+
+  const venueId = useVenueId();
+  const history = useHistory();
+
+  const handleClick = () => {
```

Comment:
It will either enter section or show a modal with an error. Any suggestions on how to name it? `enterSection` doesn't work because it does not only that

Reply:
renamed to onSectionEnter since we use `enterSection` inside + it can actually abort, if the section is full